agent-bootstrapper/src/com/thoughtworks/go/agent/bootstrapper/LauncherTempFileHandler.java

```diff
@@ -57,8 +56,9 @@ private void reapFiles() {
             for (String fileName : fileSet) {
                 File file = new File(fileName);
                 FileUtils.deleteQuietly(file);
-                JarUtil.cleanup(fileName);
-                if (!file.exists() && !JarUtil.tempFileExist(fileName)) {
+                File depsDir = new File("data", fileName);
```

Comment:
There are too any places that we use the `data` folder and refer to it via the string. Can we instead create a constant atleast on the agent side as a part of this PR. Its difficult to see where all we use the data folder. The only way to do so is to look up for the string `"data"` and that gives back way too many results.